Sources/OpenAI/Public/Models/ChatQuery.swift

```diff
@@ -119,6 +120,8 @@ public struct ChatQuery: Equatable, Codable, Streamable {
             switch self {
             case .system(let systemMessage):
                 return Self.UserMessageParam.Content.string(systemMessage.content)
+            case .developer(let developerMessage):
+                return Self.UserMessageParam.Content.string(developerMessage.content)
```

Comment:
Looks strange that we map unrelated content to `UserMessageParam.Content`, but I see that it has been implemented this way for other cases, so this is a question for another day